keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -483,7 +483,8 @@ _This section must be completed when targeting alpha to a release._
 
 * **How can this feature be enabled / disabled in a live cluster?**
   - [X] Feature gate (also fill in values in `kep.yaml`)
-    - Feature gate name: `SELinuxMountReadWriteOncePod`
+    - Feature gate name: `SELinuxMountReadWriteOncePod` (beta in 1.27)
+    - Feature gate name: `SELinuxMount` (alpha in 1.30)
```

Comment:
Do these two feature gates have any dependency on each other or are they completely independently of each other?

Reply:
That's a good question. I checked what we did with `CSIMigration` and say `CSIMigrationGCE` feature gates that depended on it, both had to be specified for GCE CSI migration to work:
https://github.com/kubernetes/kubernetes/blob/release-1.22/pkg/volume/csi/csi_plugin.go#L220-L220

I updated the KEP accordingly.